sample/jobs/sampleAppBuilder.groovy

```diff
@@ -0,0 +1,80 @@
+// map from IDA to its corresponding edx repository; "" if no corresponding repository
+final Map APPS_TO_REPO = [analytics_api: 'edx-analytics-data-api', credentials: 'credentials' , discovery: 'course-discovery', ecommerce: 'ecommerce', edxapp: 'edx-platform', 
+forum: 'cs_comments_service', harstorage: 'harstorage', insights: 'edx-analytics-dashboard', jenkins_analytics: '', nginx: '', rabbitmq: '', xqueue: 'xqueue', 
+xqwatcher: 'xqueue-watcher']
+
+// map from IDA to its main branch
+final Map APPS_TO_BRANCH = [analytics_api: 'master', credentials: 'master', discovery: 'master', ecommerce: 'master', edxapp: 'release', forum: 'master', harstorage: 'master', 
+insights: 'master', jenkins_analytics: '', nginx: '', rabbitmq: '', xqueue: 'master', xqwatcher: 'master']
+
+final String CONFIGURATION_REPO_URL = 'https://github.com/edx/configuration.git'
+final String CONFIGURATION_BRANCH = 'master'
+
+final String EDX_REPO_ROOT = 'https://github.com/edx/'
+
+// ID of credentials set up in Jenkins to log into DockerHub
+final String JENKINS_CREDENTIALS_ID = '595676cc-61ce-41d1-b3c2-ee8578540650'
```

Comment:
Will you add a TODO here that the seed job should create this and use a user-entered ID (rather than a UUID) so that this can be referred to more easily in the job.